journal/modernized/33-testimonies.adoc

```diff
@@ -82,9 +82,7 @@ and we pray, that it may please the Almighty,
 to raise up others to supply his place and service in the churches of Christ.
 
 Signed by order, in behalf of the said meeting, by Robert Hoope, Alexander Seaton.
-
-[.asterism]
-'''
+//Does this need any signed signature stuff?
 
 === The Testimony of Friends of Leinster province, concerning William Edmundson.
```

Comment:
For testimonies, I always make the headings blurbs, like this:
```ASCIIDOC
[.blurb]
=== The Testimony of blah blah blah
```
I think it looks a little nicer having these centered and italicized instead of left-justified and bold.  Just a personal preference, but I've done it this way in a bunch of other ones, so we should probably be consistent. 👍 So you'll have to scan back through the asciidoc looking for the === headings, and apply the [.blurb] class to them all.

Reply:
Sounds good! I really do look at the Style Guide frequently, and it's easier since I'm getting more familiarized with it, but i sure do miss a lot. :)